docs/5.x/changelog.md

```diff
@@ -21,6 +21,22 @@ LTS starts with `3.51.0` with minor backwards compatible changes added over time
 
 ## 5.x Releases
 
+### 5.2.0-rc.3
+
+#### Bugfixes
+
+* Fix an issue with deploying AWS clusters using provisioner.
+* Fix an issue with installers downloaded from the distribution portal.
+* Fix an issue with expanding clusters installed via an Ops Center.
+* Fix an issue with garbage collection not working on older clusters sometimes.
+
+### 5.2.0-rc.2
+
+#### Bugfixes
+
+* Fix `tele build` failure when encountering unrecognized resources.
+* Fix an issue with garbage collector removing used packages sometimes.
```

Comment:
ditto here: it was kind of related to only failing on older clusters (where it was not supposed to be used in the first place).